packages/engine-server/src/drivers/file/storev2.ts

```diff
@@ -446,6 +447,34 @@ export class FileStorage implements DStore {
             errors.push(err);
             return;
           }
+          if (this.engine.config.dev?.enableUnrefLinks) {
+            try {
+              let ctx = "findUnreferencedLinks";
+              let duration;
+              const start = process.hrtime();
+
+              const unreferencedLinks = LinkUtils.findUnreferencedLinks({
```

Comment:
the result of `findLinks` is set to `cacheUpdates[n.fname].data.links` before this and I'm adding the unref links to that. So it will be added to the cache every time, but only after it was overwritten with the result of `findLinks`